lib/trento_web/views/v1/health_overview_view.ex

```diff
@@ -37,12 +48,21 @@ defmodule TrentoWeb.V1.HealthOverviewView do
   defp extract_database_id([%DatabaseInstanceReadModel{sap_system_id: sap_system_id} | _]),
     do: sap_system_id
 
-  @spec extract_cluster_id([DatabaseInstanceReadModel.t()]) :: String.t()
+  @spec extract_cluster_id([ApplicationInstanceReadModel.t() | DatabaseInstanceReadModel.t()]) ::
```

Comment:
Isn't the spec `@spec extract_cluster_id([ApplicationInstanceReadModel.t() | DatabaseInstanceReadModel.t()]) :: String.t() | nil`?

Reply:
Yes!